java/velox4j/src/main/cpp/main/velox4j/arrow/Arrow.cpp

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
I wonder why we need the `java` top-level directory because it looks a bit strange to have cpp file under it. Can we just start with `velox4j`?

Reply:
There's a subdirectory `test` under python, while under `java` there's only the `velox4j` folder which might make the layer appear unnecessary. I wonder if we can follow the similar structure as `python`.